### PR TITLE
Dandelion Sprout's Serbo-Croatian List

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/FiltersRegistry/issues/591

### CODE REVIEW

File: filters/ThirdParty/filter_252_DandelionSproutSerbian/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "For the first time in known history, this is a filterlist for websites in Serbian, Montenegrin, Croatian and Bosnian. This means that all significant language groups in Europe now have filterlists for them.",
+  "timeAdded": 1647852134737,
+  "homepage": "https://github.com/DandelionSprout/adfilt",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 7,
+  "subscriptionUrl": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/SerboCroatianList.txt",
+  "tags": [
+    "purpose:ads",
+    "lang:sr"
```
Comment (ameshkov) [inline filters/ThirdParty/filter_252_DandelionSproutSerbian/metadata.json:13]: We'll need `lang:hr` as well.

Also, don't forget that the tag definition must be added as well.
Please try building the registry to see if there're missing tags.

**Review by ameshkov — CHANGES_REQUESTED** [review]

**Comment by slavaleleka** [comment]
why the directory named just `filter_252_DandelionSproutSerbian`? 
seems like it should be `filter_252_DandelionSproutSerboCroatian`

**Comment by slavaleleka** [comment]
we also need translations for this filter, but let's add it in another PR

**Comment by ameshkov** [comment]
```
Writing filters metadata
Missing tag with keyword: lang:sr
Missing tag with keyword: lang:hr
/home/runner/work/FiltersRegistry/FiltersRegistry/node_modules/adguard-filters-compiler/index.js:18
    throw error;
    ^

Error: Missing tag with keyword: lang:sr, lang:hr
    at replaceTagKeywords (/home/runner/work/FiltersRegistry/FiltersRegistry/node_modules/adguard-filters-compiler/src/main/platforms/generator.js:224:19)
    at writeFiltersMetadata (/home/runner/work/FiltersRegistry/FiltersRegistry/node_modules/adguard-filters-compiler/src/main/platforms/generator.js:480:27)
```

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**